### ISSUE TITLE
No password works - possibly because of MS Account?

### ISSUE THREAD

**Issue by ktos** [PowerShell/Win32-OpenSSH#390]
I am using Windows 10 Anniversary Update and using MS Account (only, never set-up local account on this machine). Started `sshd.exe` from the latest release and tried to log-in: no known for me usernames/passwords are working.

I changed my MS Account password recently and tried that password and probably two ones before. Tried both username which is working in SMB (which is in the %USERPROFILE% folder name) and username derived from the first part of my e-mail.

In logs I can see that username from the first part of e-mail is invalid:
```
1612 00:29:20 650 Failed none for invalid user ktos from :: port 65019 ssh2
1612 00:29:20 650 Failed keyboard-interactive for invalid user ktos from :: port 65019 ssh2
1612 00:29:23 448 Failed password for invalid user ktos from :: port 65019 ssh2
```

But for the "normal" username there is just failed password:
```
13724 00:28:59 481 Failed password for Marcin from :: port 65016 ssh2
13724 00:29:02 865 Failed password for Marcin from :: port 65016 ssh2
13724 00:29:05 945 Failed password for Marcin from :: port 65016 ssh2
```

I tried every password I ever set on this machine probably and no password works. Is this another MSAccount-related scenario (like SMB, SQLLocalDB and more?).

**Comment by ddwolf** [comment]
for me, I have a valid user named usera in my windows machine, but the sshd keeps print 
"Invalid user usera from x.x.x.x port 34777"
"Failed none for invalid user usera from x.x.x.x port 34777 ssh2"
...

how to make usera a valid user for sshd?

**Comment by bagajjal** [comment]
@ddwolf - please share the sshd.log with DEBUG3 enabled. 
If you are using a domain account then login like this, ssh user@domain@host.
For more information,
https://github.com/PowerShell/Win32-OpenSSH/wiki/ssh.exe-examples

**Comment by ddwolf** [comment]
@bagajjal I have found the problem: i made a typo in my username.

**Comment by jebej** [comment]
Hello, reviving and old thread, but I am also having trouble logging in.

I double check I had no typos in my user name, but still get the invalid user message from the event log:

```
sshd: Invalid user jbejanin from 172.16.37.64 port 1434
```

I started the server in debug3 mode, and singled out this interesting part:

```
debug3: get_passwd: Invalid account type: 3.
Invalid user jbejanin from 172.16.37.64 port 1446
```

**Comment by manojampalam** [comment]
Looks like jbejanin is getting resolved to a domain. Are you sure there is no domain accessible by that name?

**Comment by jebej** [comment]
The host computer name is also `jbejanin`, could that be the issue?

**Comment by manojampalam** [comment]
Yes. It could very well be the issue. Can you try jbejanin\jbejanin and check if it makes a difference? 
In any case, do open a new issue and copy these details.

**Comment by oldium** [comment]
@jebej See #1252 and a fix PowerShell/openssh-portable#395. I have same user name as the computer name. Login with COMPUTER\user now works for me.

**Comment by maertendMSFT** [comment]
This was fixed in the latest release.
